web/src/components/SearchListItem.tsx

```diff
@@ -56,12 +56,30 @@ const StyledLink = styled(Link)`
 `
 
 type SearchListItemProps = {
-  category: CategoryModel
+  title: string
   contentWithoutHtml: string
   query: string
-}
-
-const SearchListItem = ({ category, contentWithoutHtml, query }: SearchListItemProps): ReactElement => {
+} & (
+  | {
+      path: string
+      thumbnail?: string
+      url?: never
+    }
+  | {
+      url: string
+      path?: never
+      thumbnail?: never
+    }
+)
```

Comment:
This is duplicated, can we use the exported type from `useAllPossibleSearchResults` instead?

Reply:
It was only this complicated because of the offers, removing the offers fixed this.